.github/workflows/publish_sdk.yml

```diff
@@ -0,0 +1,27 @@
+name: Prepare release of SDK
```

Comment:
(Please do not fix now)

We probably need to think more about proper terminology here.

What should we call "an SDK" in our context? What does Pulumi do for other languagues? Is SDK: 

- a provider-specific library (e.g. Google Native) 

or

- the whole project

or 

- the common library (created in this PR: `org.virtuslab:pulumi-kotlin`)

I'm mentioning that, because it's used in a couple of contexts (e.g. README.md first heading). It would be nice to stay consistent with the way community calls things (I'm not sure what tihs means for us yet).

<img width="934" alt="Screenshot 2023-08-30 at 10 36 31" src="https://github.com/VirtuslabRnD/pulumi-kotlin/assets/4415632/c17be470-6665-4fac-9ed5-a0396c5f10b1">